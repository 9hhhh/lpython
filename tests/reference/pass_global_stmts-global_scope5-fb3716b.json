{
    "basename": "pass_global_stmts-global_scope5-fb3716b",
    "cmd": "lfortran --pass=global_stmts --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/global_scope5.f90",
    "infile_hash": "63e512d3fe5d83aa6195002b9c31ddc77e1a93ac37df38989f782ed7",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "pass_global_stmts-global_scope5-fb3716b.stdout",
    "stdout_hash": "ddf2d454c90d54e88c6a88290136887745567a7f0bc72b2fb2eafec0",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}