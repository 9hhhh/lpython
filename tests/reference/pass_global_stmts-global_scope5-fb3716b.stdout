(TranslationUnit (SymbolTable 58a7bc47 {f: (Function (SymbolTable 495ede5c {f0: (Variable 495ede5c f0 0 (Integer 4 [])), f1: (Variable 495ede5c f1 0 (Integer 4 [])), f2: (Variable 495ede5c f2 4 (Integer 4 []))}) f [] [(= (Var 58a7bc47 x) (Num 6 (Integer 4 []))) (= (Var 495ede5c f1) (BinOp (Num 2 (Integer 4 [])) operatorType2 (Var 58a7bc47 x) (Integer 4 []))) (= (Var 58a7bc47 x) (BinOp (Var 58a7bc47 x) operatorType0 (Num 1 (Integer 4 [])) (Integer 4 []))) (= (Var 495ede5c f2) (BinOp (Num 3 (Integer 4 [])) operatorType2 (Var 58a7bc47 x) (Integer 4 [])))] () (Var 495ede5c f2) ()), x: (Variable 58a7bc47 x 0 (Integer 4 []))}) [])
